pwiz_tools/Skyline/Model/Databinding/Entities/ColumnToolTips.resx

```diff
@@ -548,6 +548,10 @@ The options for determining this value can be specified on the Quantification ta
   <data name="MaxBestRetentionTime">
     <value>Maximum of the precursor BestRetentionTime values</value>
   </data>
+  <data name="MaxDetectionQValue" xml:space="preserve">
+    <value>A  maximum of the false discovery rate (FDR) score assigned to each chosen target peak after applying a mPro

```

Comment:
How about:
"The median false discovery rate (FDR) assigned to the chosen peak for this peptide across all of the replicates."

FDR values usually come from mProphet models, but they can also come from SWATH or EncyclopeDIA search results. (Although I think EnyclopeDIA always assigns the same q-value to each replicate so these summary values are not as useful).

Reply:
I copied the description from the Detection Q Value and I think it's better to be consistent. Also the current description points to the origin of the q values, which is helpful if a user would be wondering why they are null.